submissions/peterIyiola/css/contact.css

```diff
@@ -214,5 +214,4 @@ footer    {
 
 
 }
-    
-
+   
```

Comment:
You ought not to tamper with someone else file. So, revert it back to the way it was. Add the lines you removed.

Reply:
@ainobs You need to fix this